### PR TITLE
[DOC] Add gcp dataproc gpu limit [skip ci]

### LINKED ISSUE #6747: [DOC] Add info in GCP Dataproc getting started guide for how to handle scenario when no GPUs are available
Current GCP Dataproc getting started steps have a few cluster creation commands: https://nvidia.github.io/spark-rapids/docs/get-started/getting-started-gcp.html.  A few feedback items:

- [ ] It is possible that the user may encounter an issue where no GPUs are available in the region/zone they are attempting to create a cluster due to account limits or overall GCP capacity limits.  We should add explicit information or a pointer to GCP docs on how to check account limits and/or try another region or zone that would have GPUs available.
- [ ] Explain parameters that are being used such number of GPUs, executors
- [ ] Update default cluster config: Use 4 workers/2xT4 and 4 worker/1xA100

### PR DESCRIPTION
**Pull request by mattahrens**
Closes #6747 

Adding additional docs for cluster creation info with troubleshooting info.  Also updated user tools package to official name to be used in pip deployment.

### CODE REVIEW

**Comment by mattahrens** [comment]
build

File: docs/get-started/getting-started-gcp.md [hunk 1]
```diff
@@ -244,13 +252,20 @@ metadata parameter `MIG_CGI`. Below is an example of using a profile name and a
...
 
 If you'd like to further accelerate init time to 4-5 minutes, create a custom Dataproc image using
 [this](#build-custom-dataproc-image-to-accelerate-cluster-init-time) guide. 
 
+### Cluster creation troubleshooting
+If you encounter an error related to GPUs not being available because of your account quotas, please 
+go to this page for updating your quotas: [Quotas and limits](https://cloud.google.com/compute/quotas).
+
+If you encounter an error related to GPUs not available in the specific region or zone, you will 
+need to update the REGION or ZONE parameter in the cluster creation command.
```
Comment (tgravescs) [inline docs/get-started/getting-started-gcp.md:267]: is there any case they are temporarily unavailable and we should state try back later?  I realize that isn't ideal though.

Comment (mattahrens) [reply docs/get-started/getting-started-gcp.md:267]: It is possible and GCP error message could state that explicitly, e.g. 
```Operation type [insert] failed with message "The zone 'projects/xxxxxx/zones/us-central1-a' does not have enough resources available to fulfill the request. Try a different zone, or try again later." ```

I'd rather point to updating REGION or ZONE as that is the specific limit issue that I've seen in practice on Dataproc.

File: docs/get-started/getting-started-gcp.md [hunk 2]
```diff
@@ -197,6 +197,10 @@ gcloud dataproc clusters create $CLUSTER_NAME  \
     --subnet=default
 ```
 
+Explanation of parameters:
+* NUM_GPUS = number of GPUs to attach to each worker nodes in the cluster
+* NUM_WORKERS = number of Spark worer nodes in the cluster
```
[the remark this answers is not in the document]
Comment (mattahrens) [reply docs/get-started/getting-started-gcp.md]: Fixed in latest commit

File: docs/get-started/getting-started-gcp.md [hunk 3]
```diff
@@ -236,6 +240,10 @@ gcloud dataproc clusters create $CLUSTER_NAME  \
     --subnet=default
 ``` 
 
+Explanation of parameters:
+* NUM_GPUS = number of GPUs to attach to each worker nodes in the cluster
+* NUM_WORKERS = number of Spark worer nodes in the cluster
```
[the remark this answers is not in the document]
Comment (mattahrens) [reply docs/get-started/getting-started-gcp.md]: Fixed in latest commit

**Comment by mattahrens** [comment]
build

**Review by tgravescs — APPROVED** [review]

**Pull request merged**